website/docs/r/datapipeline.html.markdown

```diff
@@ -0,0 +1,43 @@
+---
+layout: "aws"
+page_title: "AWS: aws_datapipeline"
+sidebar_current: "docs-aws-resource-datapipeline"
```

Comment:
This page is missing a sidebar link in `website/aws.erb` and among the other renaming in this file:

```suggestion
sidebar_current: "docs-aws-resource-datapipeline-pipeline"
```

Reply:
fixed